docs/conf/portland/2022/news/conference-going-virtual.rst

```diff
@@ -0,0 +1,111 @@
+We'll explain a bit more below the reasoning for this decision,
+but the high-level issues are:
+
+* **After talking with our vendors and staff, we determined we couldn't produce the quality of event that our attendees expect if we run a hybrid event.**
+* Omicron has changed the risk profile associated with travel and large gatherings, and increased the number of unknown risks that we face as organizers.
+
+We are excited about the quality of our previous virtual events,
+and look forward to iterating on and improving our event once again this year.
+We have been just as excited as you at the prospect of being able to gather safely,
+but we feel the need to hold off for one more year.
+
+We're also looking at the ability for some of our meetups to host watch parties.
+The details are still being worked out,
+but we'd love to find a way for small groups of people to gather in this fashion.
+
+Updated conference details
+--------------------------
+
+Our 2022 conference will be structured the same to our previous virtual events:
+
+* Sunday, May 22: Writing Day
+* Monday, May 23: Conference talks, Lightning Talks, Unconference, Social event
+* Tuesday, May 24: Conference talks, Job Fair, Lightning Talks, Unconference
+
+We're excited to experiment and improve on our existing format as well.
+One change we're considering is doing scheduled Q&A blocks with 2 speakers,
+so attendees don't feel rushed going from the main stage to the Q&A after every talk.
+
+Now that we have made this decision,
+we are moving on to the other tasks of launching the conference.
+We expect the full conference website and ticket sales to be launched in the next couple weeks.
+The `sponsorship prospectus <https://www.writethedocs.org/conf/portland/2022/sponsors/prospectus/>`_ is now available as well.
+
+Reasons for going virtual
+-------------------------
+
+There are four major areas that formed the basis of our decision:
+
+* Physical health
+* Event success
+* Mental health
+* Organizational sustainability
+
+The biggest worry that we have when organizing the conference is a COVID outbreak.
+We were already planning a number of mitigations,
+but this risk is always present whenever we gather in a large group.
+Omicron has lead to the highest case counts in the USA,
+and increased our anxiety about possible negative outcomes.
+
+The quality of the event was another large factor in our decision.
+We discussed at length,
+but we couldn't find a good way to integrate in-person and virtual attendees.
+We figured out a way to handle it for talks,
+but things like the job fair, and unconference don't lend themselves to mixing between in-person and virtual.
+The biggest requirement to do this effectively is reliable in-person WiFi,
+which has *always* proven to be difficult in the past.
+We think it is perhaps possible to integrate the two spaces well in the future,
+but we weren't able to commit to it this year.
```

Comment:
The phrase in line 68 is not consistent with line 67, and I think lines 71 and 72 are TMI, unless we're implicitly committing to a hybrid event for 2023. So I suggest:

```suggestion
but had not solved the issue for the job fair and unconference. Hybrid conferences require
reliable in-person WiFi, which has *always* been a difficult issue in past conferences.
```